### DISCUSSION TITLE
Visual Studio 2022 - ScrollBar Track

### DISCUSSION THREAD

**Discussion by brunotourinho** [arcticicestudio/nord#252] [Development]
Sorry to bother you guys, I didn't find the repo for the VS2022 Theme (in preview)
The theme is working great for me so far, but one little thing annoys me,is that the left scroll bar track is invisible. 
I wish you could tell me how to make it a little grey-ish.

![nord1](https://github.com/nordtheme/visual-studio-code/assets/12257968/d1f2240b-a8b8-45c6-8432-0d019583ff86)
![nord2](https://github.com/nordtheme/visual-studio-code/assets/12257968/75ca0461-f864-40ac-9458-ce311dee7cfd)

Thanks in advance

**Comment by svengreb** [comment]
@brunotourinho Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, and being related to the project “Visual Studio Code“ project at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
